backend/open_webui/routers/knowledge.py

```
@@ -17,14 +17,15 @@
     process_files_batch,
     BatchProcessFilesForm,
 )
-
+from open_webui.storage.provider import Storage
```

Ruff (0.8.2)

`open_webui.storage.provider.Storage` imported but unused

Remove unused import: `open_webui.storage.provider.Storage`

(F401)